source/installation.txt

```diff
@@ -4,10 +4,23 @@
 Installation
 ============
 
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+These documents provide instructions to install {+c2c-full-product-name+}.
+
+:ref:`c2c-install-linux`
+   Install the official builds of {+c2c-product-name+} on Linux-based operating systems.
+
+:ref:`c2c-install-macos`
```

Comment:
[broken ref]

Looks like this ref is not resolving properly:

<img width="659" alt="Screen Shot 2022-05-25 at 2 09 01 PM" src="https://user-images.githubusercontent.com/73852296/170336404-41ef7a6b-03aa-4f2b-a278-deb3c57b4e0d.png">
.